.gitignore

```diff
@@ -0,0 +1,4 @@
+test.md
+*.yml
+.hintrc
+.stylelintrc.json
```

Comment:
- [ ] Please remove these files from the .gitignore, they are not meant to be ignored. You need them for your linters to pass.